packages/hash/api/src/db/adapter.ts

```diff
@@ -197,7 +192,7 @@ export type DbFileProperties = {
 };
 
 export type OrgInfoProvidedAtCreation = {
-  orgSize: OrgSize;
+  orgSize: any;
```

Comment:
Because `OrgSize` was a GQL defined type, which has now been deleted. This is a broken file that we will eventually delete (it's the adapter to the old database), so I didn't want to invest in a proper fix for the type if its just going to be unused and then deleted anyway. I can add a `@todo`, but not sure what purpose it serves as no one is developing in this file, and we will delete it (hopefully) soon.

Reply:
Gotcha, I missed the fact that this file was an adapter, no worries